troubleshooting/macbook-troubleshooting.md

```diff
@@ -139,3 +139,182 @@ Please see [this thread o the qubes-devel mailing list][macbook-air-2012-5-1].
 [bluetooth-replacement]: https://www.ifixit.com/Guide/MacBook+Air+13-Inch+Mid+2011+AirPort-Bluetooth+Card+Replacement/6360
 [macbook-air-2012-5-1]: https://groups.google.com/d/topic/qubes-devel/uLDYGdKk_Dk/discussion
 
+
+
+MacBook Pro Retina, 15 inch, Mid-2015 (MacBookPro 11,5)
+-------------------------------------------------------
+
+In this section, I explain how I installed Qubes 3.2 on a MacBook Pro Retina 2015 (MacBookPro 11,5).
+Good news: the relevant stuff works.
+Bad news: still some minor issue to investigate.
+
+During my nigths trying to get Qubes OS working, I faced tow main and blocking issues:
+*   no boot, due to empty xen.cfg file
+*   system freeze, due to Broadcom BCM43602 wifi card
+
+I am already using Qubes for my daily job on Intel NUC. For the time being, I installed Qubes on Macbook for test purposes. Later on I will review the security implications.
+
+This model has the following features:
+
+*	2,5 GHz Intel Core i7-4870HQ (2 quad cores)
+*	Dual Graphic Card
+    *  Intel Iris Pro
+    *  AMD Radeon R9 M370X
+*	16Gb RAM
+*	512Gb SDD
+*	Broadcom BCM43602 802.11ac wifi adapter
+
+### 1. Reclaim space to be able to multiboot OSX
+
+For security reasons, you should install Qubes using the whole disk. I preferred to keep OSX, so I shrunk OS partition:
+*  reboot in recovery mode
+*  run disk utility and shrink OSX partition, eg 150GB for OSX and the remaining space for Qubes OS
+*  reboot
+
+
+### 2. Boot installer
+
+Download and prepare a USB with Qubes 3.2
+
+You can install Qubes using BIOS or UEFI:
+*  BIOS/CSM/Legacy: I have not been able to install using legagy, but I did not spent a lot of time on it.
+   * power on the macbook pressing Option button
+   * GRUB menu appears, with option to test media & install, install, troubleshoot and rescue
+   * whatever option I choose, I have a quick flash and back to the menu
+*  UEFI, using rEFInd: I have been successful, despite some issues to be fixed manually, after installation completion
+   * download [rEFInd] ZIP file, unzip it and run installer
+   * if installation fails due to SIP, reboot in recovery mode, open a terminal and issue command
+   ~~~
+   crsutil disable
+   ~~~
+   * reboot and you will see some icons
+   * choose Boot EFI\BOOT\xen.efi from ANACONDA
+   * after a will the graphical installer is up and running, with keyboard and touchpad working
+
+### 3. Installation
+
+*  As a general rule, keep the default values proposed during installation: you will change them later on
+*  Keep English, as language, locale
+*  My macbook has a US keyboard, so I cannot say what happens if you change keyboard layout
+*  DO NOT CHANGE the timezone, because it will trigger the wifi card, leading to a system freeze
```

Comment:
Ough... if that's really the case, could you create an issue in qubes-issues for that? Or maybe you already done that?